src/keystats/keystatsLabel.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+
+export const StatsLabel:React.SFC = ({ children }) => <span className="label label--small">{children}: </span>
+
+export const StatsValue:React.SFC = ({ children }) => <span className="label">{children}</span>
```

Comment:
You could perhaps extract the two functions above with the same definition into a single function but called with className as an argument as well? For example: StatsByClassName:React.SFC = ({children}, className) => etc...,

Reply:
That would mean that the calling side would need to know implementation details of the component, in this case CSS class names used. I'd rather have 10 specialised components and not have to know any implementation details.

Alternatively, you can have an internal method that those two use, e.g.:

```typescript
import * as classnames from 'classnames';
import * as React from 'react';
import { Component, ReactNode } from 'react';

export class Label {
    public static defaultProps: Partial<LabelProps> = {
        small: false,
    };

    public render(): ReactNode {
        const { children, small } = this.props;

        const className = classnames('label', {
            'label--small': small,
        });

        <span className={className}>{children}</span>
    }
}

interface LabelProps {
    small?: boolean;
}
```

And then use it as follows:

```jsx
<Label small={true}>Something</Label>
<Label>Larger</Label>
```